module/documents/item.mjs

```diff
@@ -2128,7 +2128,12 @@ export default class Item5e extends Item {
   _onCreateOwnedWeapon(data, isNPC) {
 
     // NPCs automatically equip items and are proficient with them
-    if ( isNPC ) return {"system.equipped": true, "system.proficient": true};
+    if ( isNPC ) {
+      const updates = {};
+      if ( !foundry.utils.getProperty(data, "system.equipped") ) updates["system.equipped"] = true;
```

Comment:
These checks need to be `hasProperty` instead of `getProperty` because the value may be `false`

Reply:
🤦 fixed